tests/apiserver-proxy-tests/apiserver-proxy/internal/client.go

```diff
@@ -0,0 +1,18 @@
+package graphql
```

Comment:
Folder is called `internal`, file `client.go` and package `graphql`. Please make up your mind ;)